extide/libs.gradle/manifest.mf

```diff
@@ -2,4 +2,4 @@ Manifest-Version: 1.0
 AutoUpdate-Show-In-Client: false
 OpenIDE-Module: org.netbeans.modules.libs.gradle/8
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/libs/gradle/Bundle.properties
-OpenIDE-Module-Specification-Version: 8.1
+OpenIDE-Module-Specification-Version: 8.0.2.1
```

Comment:
This is going backwards.  We really need to think how we handle module specification versions with regard to third-party versions, as it conflicts with the automatic version increments we do after every branch.

Not sure what the best answer here is?

Reply:
My take on this is, that the Spec Version does not relate to the wrapped library. I double checked the JNA spec version and indeed, it stands at 2.15, while JNA itself stands at 5.12.1.

Reply:
we could do it where it works and makes sense and not do it where it doesn't?
e.g nb-javac uses the major version right now to sync to the JDK version it is based on, numbers after that are changed by the system (aka Neil ;))

https://github.com/apache/netbeans/blob/347d63a7b2acd3254b8bb9f6d512c89f4b84a48e/java/libs.nbjavacapi/manifest.mf#L5

Reply:
you could just wait till gradle overtakes the spec version so that it doesn't walk backwards :)